homework23/src/homework23/QueueArrayDouble

```diff
@@ -0,0 +1,115 @@
+package homework23;
```

Comment:
Please see: https://en.wikipedia.org/wiki/Queue_(abstract_data_type)

Queue (if not specified otherwise) is FIFO data structure. This means that first element added should be removed first (eg. if you are the first one in line to buy cookies, you should be the first one to be served).